### PR TITLE
docs: Document how to run e2e tests (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -197,6 +201,16 @@ If that gets executed in one of the package folders it will only run the tests
 of this package. If it gets executed in the n8n-root folder it will run all
 tests of all packages.
 
+#### E2E tests
+
+E2E tests can be started via one of the following commands:
+
+- `pnpm test:e2e:ui`: Run e2e tests interactively against port 5678
```
Comment (cstuncsik) [inline CONTRIBUTING.md:208]: Maybe we can add this after: _(runs the built UI, it does not react to editor code changes)_

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -197,6 +201,16 @@ If that gets executed in one of the package folders it will only run the tests
...
+- `pnpm test:e2e:dev`: Run e2e tests interactively against port 8080 (editor-ui dev server, needs to be running separately)
```
Comment (cstuncsik) [inline CONTRIBUTING.md:209]: The command starts a dev server automatically so you don't have to run it separately.
Maybe what's important here is it reacts to editor-ui code changes.

**Review by cstuncsik — COMMENTED** [review]
Thanks for documenting this, I just left some suggestions for more clarity of the reader

**Comment by tomi** [comment]
@cstuncsik thanks for the comments, I understand it better myself now. I clarified the text. Let me know what you think!

**Comment by cstuncsik** [comment]
Nice! All good

**Review by cstuncsik — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.19.4`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.19.4)

**Pull request merged**